charts/fluent-operator/values.yaml

```diff
@@ -1,84 +1,65 @@
+  # - name: "image-pull-secret"
+  logPath:
+    crio: /var/log
+    containerd: /var/log
 
 fluentbit:
   image: "kubesphere/fluent-bit"
   tag: "v1.8.3"
-
-## Reference to one or more secrets to be used when pulling images
-## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
-##
-imagePullSecrets: []
-# - name: "image-pull-secret"
-nameOverride: ""
-fullnameOverride: ""
-namespaceOverride: ""
-
-resources:
   # fluentbit resources. If you do want to specify resources, adjust them as necessary
-  fluentbit:
-    # Fluent Bit resources. User can adjust it based on the log volume.
-    resources:
-      limits:
-        cpu: 500m
-        memory: 200Mi
-      requests:
-        cpu: 10m
-        memory: 25Mi
-
-  operator:
-  # FluentBit operator resources. Usually user needn't to adjust these.
-    resources:
-      limits:
-        cpu: 100m
-        memory: 30Mi
-      requests:
-        cpu: 100m
-        memory: 20Mi
-
-secrets: []
-
-Kubernetes: false
-
-#Set a limit of memory that Tail plugin can use when appending data to the Engine.
-#If the limit is reach, it will be paused; when the data is flushed it resumes.
-#if the inbound traffic is less than 2.4Mbps, setting memBufLimit to 5MB is enough
-#if the inbound traffic is less than 4.0Mbps, setting memBufLimit to 10MB is enough
-#if the inbound traffic is less than 13.64Mbps, setting memBufLimit to 50MB is enough
-input:
-  tail:
-    memBufLimit: 5MB
-#Set this to containerd or crio if you want to collect CRI format logs
-containerRuntime: docker
-logPath:
-  crio: /var/log
-  containerd: /var/log
-
-flentd:
-  name: flentd
-  port: 24224
-  image: "kubesphere/fluentd"
-  tag: "v1.14.4"
-  replicas: 1
-  watchedNamespaces:
-    - kube-system
-    - kubesphere-monitoring-system
-
-
-output:
+  #You can adjust it based on the log volume.
+  resources:
+    limits:
+      cpu: 500m
+      memory: 200Mi
+    requests:
+      cpu: 10m
+      memory: 25Mi
+  ## Reference to one or more secrets to be used when pulling images
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+  ##
+  imagePullSecrets: [ ]
+  # - name: "image-pull-secret"
+  secrets: []
+  #Set a limit of memory that Tail plugin can use when appending data to the Engine.
+  #If the limit is reach, it will be paused; when the data is flushed it resumes.
+  #if the inbound traffic is less than 2.4Mbps, setting memBufLimit to 5MB is enough
+  #if the inbound traffic is less than 4.0Mbps, setting memBufLimit to 10MB is enough
+  #if the inbound traffic is less than 13.64Mbps, setting memBufLimit to 50MB is enough
+  input:
+    tail:
+      memBufLimit: 5MB
   #Configure the output plugin parameter in FluentBit.
   #You can set enable to true to output logs to the specified location.
-  fluentbit:
+  output:
     forward:
-      enable: false
-      host: "fluentd.kubesphere-logging-system.svc"
+      host: "kubesphere-logging-system.svc"
```

Comment:
I think host and port should be removed and only defined in fluentd part. I think we should keep host and port defined in one place is enough. Otherwise, forward.enabled to enable or disable the function of fluentd is enough.

Reply:
What do u guys think of? @benjaminhuo @wenchajun

Reply:
forward logs to fluentd should be controlled by the fluentd switch, and should be in the fluentd part.

Reply:
Fluent Bit also needs an output forward plugin. Do you mean to use double switches to control it?

Reply:
Yeah, I almost get it. Fluentd can enable by its switch for other requirements. And forward to fluentd would use two switches.

Reply:
My understanding is that the `forward` profile is controlled by `fluentd.enable` and no dedicated forward profile is being set up. Is this right?

Reply:
If suggested to remove Fluentbit forward part and needs some docs to implement this case.  User's confusion about the forward plugin can be defined to other fluentds. For other cases, one fluentd activated, there must be one service, the docs also should implement this.

Reply:
fluentbit forwarding to fluentd is only required when fluentd is enabled.
No need to add fluentbit forward CR in fluentbit configs because you don't know if the fluentd is enabled in fluentbit config.